frontend/src/pages/profile/components/achievement/achievement.tsx

```diff
@@ -1,29 +1,62 @@
-import { type FC } from 'react';
-
+import { ReactComponent as ArticlesAchievementIcon } from '~/assets/img/articles-achievement.svg';
+import { ReactComponent as CommentsAchievementIcon } from '~/assets/img/comments-achievement.svg';
 import { getValidClassNames } from '~/libs/helpers/helpers.js';
 import { type AchievementWithProgressResponseDto } from '~/packages/achievements/achievements.js';
-import { getProgressStyleClass } from '~/pages/profile/libs/helpers/helpers.js';
 
+import { AchievementItemConfig } from '../../libs/enums/enums.js';
+import { getRadialProgressStyleString } from '../../libs/helpers/helpers.js';
 import styles from './styles.module.scss';
 
 type Properties = {
   achievement: AchievementWithProgressResponseDto;
   className?: string;
 };
 
-const Achievement: FC<Properties> = ({ achievement, className }) => {
-  const { name, progress } = achievement;
-  const statusStyleClass = getProgressStyleClass(progress);
+const Achievement: React.FC<Properties> = ({ achievement }) => {
+  const { name, referenceTable, progress } = achievement;
+  const achievementEntityClassName: string | undefined = {
+    'comments': styles.comments,
+    'articles': styles.articles,
+  }[referenceTable];
+
+  const AchievementImage: React.FC<React.SVGProps<SVGSVGElement>> | undefined =
+    {
+      'comments': CommentsAchievementIcon,
+      'articles': ArticlesAchievementIcon,
+    }[referenceTable];
 
   return (
-    <div
-      className={getValidClassNames(
-        styles.achievementItem,
-        className,
-        styles[statusStyleClass],
-      )}
-    >
-      <h4>{name}</h4>
+    <div className={styles.achievementWrapper}>
+      <div
+        className={getValidClassNames(
+          styles.achievementBadge,
+          achievementEntityClassName,
+          !progress && styles.notStarted,
+        )}
+      >
+        <svg viewBox="0 0 100 100" className={styles.progressWrapper}>
+          <circle
+            cx="50"
+            cy="50"
+            className={styles.progressTrack}
+            r={AchievementItemConfig.PROGRESS_RADIUS}
+          />
+          <circle
+            cx="50"
+            cy="50"
+            className={styles.progress}
+            r={AchievementItemConfig.PROGRESS_RADIUS}
+            strokeDasharray={getRadialProgressStyleString(
+              progress,
+              AchievementItemConfig.PROGRESS_RADIUS,
+            )}
+          />
+        </svg>
```

Comment:
Can we add progress colors for this circle?
Like
Green - for `100%` **|** Orange\Yellow - for  `progress > 30%` **|** Gray - for `progress < 30%`